Modules/Exercise04/Tests/Unit/Controllers/CalendarControllerTest.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+namespace Modules\Exercise04\Tests\Unit\Controllers;
+
+use Mockery\MockInterface;
+use Modules\Exercise04\Http\Controllers\CalendarController;
+use Modules\Exercise04\Services\CalendarService;
+use Tests\TestCase;
+
+class CalendarControllerTest extends TestCase
+{
+    public function test_index()
+    {
+        $calendarService = $this->mock(CalendarService::class, function (MockInterface $mock) {
```

Comment:
Đoạn này có thể viết như này cũng được, nhưng nên viết kiểu
`$this->mock(CalendarService::class)->shouldReceive(....)->andReturn(....)`

Tuy nhiên thì đoạn mock này mình không nên chỉ `shouldReceive` và `andReturn` luôn.
Mình nên làm để biết thêm là cái method `getDateClass` gọi bao nhiêu lần và những tham số truyền vào là gì